code/__DEFINES/construction.dm

```diff
@@ -32,6 +32,11 @@
 #define RWINDOW_BOLTS_HEATED 7
 #define RWINDOW_SECURE 8
 
+//reinforced plating deconstruction states
+#define PLATE_INTACT 0
```

Comment:
If these are only used in the reinforced plating file, only define them in there